.gitignore

```diff
@@ -5,7 +5,6 @@
 **/public/
 **/coverage/
 **/node_modules/
-models/examples/**/PrepareRequest-FhirMessageUnsigned.json
```

Comment:
Remove gitignore for prepare requests so prepare examples are viewable/downloadable from github